cmds/core/ls/ls.go

```diff
@@ -34,16 +34,17 @@ import (
 	"github.com/u-root/u-root/pkg/ls"
 )
 
-var (
-	all       = flag.BoolP("all", "a", false, "show hidden files")
-	human     = flag.BoolP("human-readable", "h", false, "human readable sizes")
-	directory = flag.BoolP("directory", "d", false, "list directories but not their contents")
-	long      = flag.BoolP("long", "l", false, "long form")
-	quoted    = flag.BoolP("quote-name", "Q", false, "quoted")
-	recurse   = flag.BoolP("recursive", "R", false, "equivalent to findutil's find")
-	classify  = flag.BoolP("classify", "F", false, "append indicator (one of */=>@|) to entries")
-	size      = flag.BoolP("size", "S", false, "sort by size")
-)
+type cmd struct {
```

Comment:
the comment for available options are slightly out of sync.

Reply:
I don't know. it was a convention when we started this project, but if you think it makes sense to remove them, I would agree.